package.json

```diff
@@ -27,6 +27,7 @@
   },
   "homepage": "https://github.com/codecov/uploader#readme",
   "dependencies": {
+    "fast-glob": "^3.2.7",
```

Comment:
Nice!
Wasn't it you though that said BDL's libs were perfect? :trollface:

Reply:
Haha, yeah. I figured if this was going to be a speed improvement it would be a *speed improvement*. I believe it uses good ol' minimatch under the hood, however.

Reply:
I think I saw `micromatch`, but I've already forgot. Code bits like https://github.com/codecov/uploader/issues/251 are making me question why I woke up at 4am today